src/app/athletes/athletes.component.ts

```diff
@@ -23,21 +24,27 @@ export class AthletesComponent implements OnInit {
   public selectedModalityId: number = this.allRecords;
   public selectedSportId: number = this.allRecords;
 
+  public sportsList: any;
+  public maxAthletesByPage = 6;
+
   constructor(
     private athletesService: AthletesService,
     private toastrService: ToastrService,
-    private modalitiesService: ModalitiesService
+    private modalitiesService: ModalitiesService,
+    private sportsService: SportsService
   ) {}
 
   ngOnInit() {
     const firstPage = 1;
-    const maxAthletesByPage = 6;
     this.athletesService.getAthletesByPagination(firstPage).subscribe(data => {
       this.athletes = data.results;
       this.data = data;
```

Comment:
Este `this.data` me doy cuento que no tiene uso en el HTML, o se usa en otro lado ?